style/gallery.less

```diff
@@ -18,15 +19,34 @@
 		align-items: center;
 		justify-content: center;
 		flex-direction: column;
-		height: calc( 100vh - @softkeysHeight );
 
 		img {
-			max-width: 100%;
 			background-color: #fff;
```

Comment:
this should be removed? I can still see some image with #fff background

Reply:
We need to have the white background underneath images to accommodate for transparent svg images (https://phabricator.wikimedia.org/T236302#5845706)